### PR TITLE
Logical logging: add-domain

### PR DESCRIPTION
**Pull request by Heulitig**
Continued in PR: <https://github.com/FifthTry/dotcom/pull/13>

### CODE REVIEW

**Comment by amitu** [comment]
Remove the `frontend` folder, it has been moved to `dotcom` repo.

File: Cargo.toml [hunk 1]
```diff
@@ -22,8 +22,10 @@ homepage = "https://www.fifthtry.com"
 
 
 [workspace.dependencies]
-ft-sdk = "0.1.1"
-ft-sys = "0.1.1"
+# ft-sdk = "0.1.1"
+# ft-sys = "0.1.1"
+ft-sdk = { path = "../ft-sdk/ft-sdk" } # local testing
+ft-sys = { path = "../ft-sdk/ft-sys" } # local testing
```
Comment (amitu) [inline Cargo.toml:28]: Nopes, have we done any changes to these folders? We are struct about using published and documented versions of these crates.

**Comment by amitu** [comment]
This is dotcom repo, why have we done so many frontend changes to implement logging? Close this PR and create a new clear repo with only logging changes.

**Comment by Heulitig** [comment]
Continued in PR: <https://github.com/FifthTry/dotcom/pull/13>

**Pull request closed without merging**